app/src/main/java/com/nextcloud/talk/chat/data/network/OfflineFirstChatRepository.kt

```diff
@@ -292,6 +295,24 @@ class OfflineFirstChatRepository @Inject constructor(
             updateUiForLastCommonRead()
         }
 
+    override suspend fun updateRoomMessages(internalConversationId: String, limit: Int) {
+        val lastKnown = chatDao.getNewestMessageId(internalConversationId)
+
+        Log.d(TAG, "---- updateRoomMessages ------------ with lastKnown: $lastKnown")
+        val fieldMap = getFieldMap(
+            lookIntoFuture = true,
```

Comment:
lookIntoFuture = true
is wrong. It would trigger the long polling which is not wanted in this case.